frontend/packages/operator-lifecycle-manager/src/components/deprecated-operator-warnings/DeprecatedOperatorWarnings.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { Label, FormAlert, Alert, Tooltip } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { YellowExclamationTriangleIcon } from '@console/dynamic-plugin-sdk/src/api/core-api';
+import { DeprecatedOperatorWarning } from '@console/operator-lifecycle-manager/src/types';
+
+export const DeprecatedOperatorWarningBadge: React.FC<DeprecatedOperatorWarningBadge> = ({
+  deprecation,
+  className,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <Label
+      color="orange"
+      className={className}
+      icon={
+        <Tooltip content={t('olm~Deprecated: {{message}}', { message: deprecation?.message })}>
```

Comment:
Since `message` is optional, you should make the `<Tooltip>` optional.

Reply:
Wondering if the entire label shouldn't have the `<Tooltip>` around it rather than just the icon.  The icon alone makes sense for the selects.  WDYT, @tlwu2013?

Reply:
Hmm.  https://github.com/openshift/console/blob/2d4941b365a3d98eb0d73e56e8756677d52755a2/frontend/packages/gitops-plugin/src/components/details/GitOpsDetails.tsx#L108-L110 seems to work...